setup.py

```diff
@@ -138,7 +138,9 @@ def detect_modules (self):
         # Read the config.h file (usually generated by autoconf)
         if self.compiler.compiler_type == 'msvc':
             # Add special include directory for MSVC (because MSVC is special)
-            self.compiler.include_dirs.insert(0, "src/inc-msvc/")
+            # Only needed for Python 3.5 and earlier
+            if sys.version_info[:2] < (3, 6):
```

Comment:
Are we sure it is needed for Python 3.5? Was pretty sure that version of MSVC also had `stdint.h` as well. Also what about the `config.h` that is also included in that directory?